ci/infra/azure/main.tf

```diff
@@ -1,10 +1,16 @@
 provider "azurerm" {
   features {}
+  version="=2.7.0"
```

Comment:
The specific version is on purpose?

Reply:
It is lock to use version of azurerm provider which we tested.

Reply:
It might be better to use `>=` so we don't actually stop customers from using the higher version.

Reply:
changed